defaults/main.yml

```diff
@@ -33,3 +33,6 @@ apache_mods_enabled:
   - rewrite.load
   - ssl.load
 apache_mods_disabled: []
+
+# Should apache service be started on task end
+apache_start_service: true
```

Comment:
Let's call this `apache_state` with recommended values of `started` or `stopped`.